frontend/src/pages/TermsOfUse.tsx

```diff
@@ -34,14 +34,14 @@ const TermsOfUse = () => {
               www.clothingloop.org and the application (together and separately
               hereinafter also referred to as the <span>"Platform"</span>) of
               and offered by The Clothing Loop (hereinafter referred to as “The
-              Clothing Loop" or <span>"we</span> or <span>"us"</span>) as part
+              Clothing Loop" or <span>"we"</span> or <span>"us"</span>) as part
               of Stichting Slow Fashion Movement, (KVK 85110701) and (b) the
-              terms of your participation in The Clothing Loop.{" "}
+              terms of your participation in The Clothing Loop.
             </li>
             <li>
-              By using the Platform, you (hereinafter also referred to as the{" "}
```

Comment:
Some of the `{" "}` is needed, if removed gonna be so (screenshot from latest preview) 
![image](https://user-images.githubusercontent.com/21377166/153762778-efbae4b2-7868-49ff-88d6-b5b63100b632.png)